### PR TITLE
Develop v2 1

### PR DESCRIPTION
**Pull request by amazzini**

### CODE REVIEW

**Review by karthik-sethuraman — APPROVED** [review]

**Review by karthik-sethuraman — CHANGES_REQUESTED** [review]
Need to include YANG, TREE and OAS files

File: YANG/tapi-photonic-media@2020-04-23.yang [hunk 1]
```diff
@@ -714,6 +748,49 @@ module tapi-photonic-media {
         }
         description "none";
     }
+    grouping media-channel-aconnectivity-service-end-point-spec {
```
Comment (arthurMll) [inline YANG/tapi-photonic-media@2020-04-23.yang]: Typo in the name: media-channel-**_a_**connectivitiy-service-end-point-spec

File: YANG/tapi-photonic-media@2020-04-23.yang [hunk 2]
```diff
@@ -292,6 +304,20 @@ module tapi-photonic-media {
...
     }
+    augment "/tapi-common:context/tapi-connectivity:connectivity-context/tapi-connectivity:connectivity-service/tapi-connectivity:end-point" {
+        container otsi-aconnectivity-service-end-point-spec {
+            uses otsi-aconnectivity-service-end-point-spec;
+            description "none";
+        }
+        description "none";
+    }
+    augment "/tapi-common:context/tapi-connectivity:connectivity-context/tapi-connectivity:connectivity-service/tapi-connectivity:end-point" {
+        container media-channel-aconnectivity-service-end-point-spec {
```
Comment (arthurMll) [inline YANG/tapi-photonic-media@2020-04-23.yang]: Typo in the name: media-channel-**a**connectivitiy-service-end-point-spec

**Comment by arthurMll** [comment]
I have reviewed the .yang .tree files and it seems they do not match. I think the tree files are generated wrong or based in an intermediate version of the YANG files.

**Comment by amazzini** [comment]
Also renamed
- OtsiAConnectivityServiceEndPointSpec as OtsiaConnectivityServiceEndPointSpec
- MediaChannelAConnectivityServiceEndPointSpec as McaConnectivityServiceEndPointSpec

**Comment by nigel-r-davis** [comment]
I have just reviewed the photonic media model and yang and have found another name issue following the OtsiAConnect... pattern. There is an OtsiGServerAdaptationPac. This comes out as otsi-gserver.... It ought to be otsig-server..., hence the uml should be OtsigServerAdaptationPac. There are also two forms, Otsia... and OtsiAssembly... in the uml. The OtsiAssemblyConnectionEndPointSpec... ought to be OtsiaConnectionEndPointSpec. 

Unfortunately we also have an longhand media-channel-assembly-spec - this may be arguable because we are using longhand media-channel everywhere else not mc, so the distinction between OTSI and MC is set in general. 

I suggest we stay with media-channel-assembly, but ought to fix the Otsig issue (as it is exactly the same pattern as the Otsia issue we have already fixed. We could consider the OtsiAssembly issue from a consistency perspective, but I am ok to leave it as is.

**Comment by nigel-r-davis** [comment]
Have just realized that the two items I commented on are both deprecated.

I have no issue with the PhotonicMedia model as it is.

**Review by nigel-r-davis — APPROVED** [review]
To the depth I have managed to review, I am OK to proceed.

**Review by arthurMll — APPROVED** [review]
Everything seems correct. Approved from my side.

**Review by karthik-sethuraman — APPROVED** [review]

**Pull request merged**